net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/OSPF.xml

```diff
@@ -36,6 +36,7 @@
                         <bgp>Border Gateway Protocol (BGP)</bgp>
                         <connected>Connected routes (directly attached subnet or host)</connected>
                         <kernel>Kernel routes (not installed via the zebra RIB)</kernel>
+                        <ospf>Open Shortest Path First (OSPF)</ospf>
```

Comment:
Can't redistribute to self ...

Reply:
I assumed it was for redistributing into different areas: it shows up as an option in FRR, so I didn't see any reason to not include it.

Reply:
After doing a bit more research, I noticed that redistribute options actually had arguments that the current configuration style doesn't support. This one in particularly is special in that it has a mandatory argument: I never thought to test that.